packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -902,7 +902,13 @@ class SliverReorderableListState extends State<SliverReorderableList> with Ticke
           newIndex = item.index;
         }
       } else {
-        if (itemStart <= proxyItemStart && proxyItemStart <= itemMiddle) {
+        if (item.index == _dragIndex!) {
+          // If end of the proxy is in not in ending half of item
+          // We don't process, because its drag item index
```

Comment:
```suggestion
          // we don't process, because its drag item index.
```
This comment looks a little bit confusing, could you help double check? "because it's the original dragged item"?

Reply:
@QuncCccccc I have pushed the required changes.